### ISSUE TITLE
Using Railsinstaller on OS/X: ri did not show core ruby library

### ISSUE THREAD

**Issue by PDegenPortnoy** [railsbridge-boston/railsbridge-boston#102]
RailsInstaller-1.0.4-osx-10.7 was used with OSX 10.7.5, installation was successful, but ri did not generate expected results. It seems that the ri documentation was showing only Rails extensions and not core library.  

Problem persisted with new terminal instance.

**Comment by houhoulis** [comment]
Agreed, very good description.  `ri Integer#upto` did not work, and `ri Integer` showed only the methods defined in the Rails class.

**Comment by decklin** [comment]
We're not setting `ri`/`rdoc` documentation inside the VM, to keep it small and because ruby-doc.org is a lot more user-friendly. If this bothers anyone please reopen.
